.prettierrc

```diff
@@ -0,0 +1,7 @@
+{
+  "printWidth": 100,
+  "semi": true,
+  "trailingComma": "es5",
+  "singleQuote": true,
+  "proseWrap": "never"
+}
```

Comment:
I added this prettier config file and also configured `husky` and `lint-staged` in `package.json` to run `prettier ` and `eslint` as pre-commit hooks.

Reply:
Let's just change that to 80 and we'll be good to go.

Reply:
@praveenpuglia I updated `printWidth` to `80`